src/PrestaShopBundle/Resources/config/services/adapter/address.yml

```diff
@@ -79,3 +79,6 @@ services:
     tags:
       - name: tactician.handler
         command: 'PrestaShop\PrestaShop\Core\Domain\Address\Command\EditCartAddressCommand'
+
+  prestashop.adapter.address.address_formatter:
```

Comment:
```suggestion
  prestashop.adapter.address.formatter:
```
It's redundant no?

Reply:
Yes but we usually respect the class naming, so since address is in the class name it should be in the service name as well